libs/api-interfaces/src/lib/aggregate-views/view-models/resources/vocabulary-list/vocabulary-list-entry.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { ITermViewModel } from '../term.view-model.interface';
+
+export interface IVocabularyListEntry<TAllowedValues> {
+    term: ITermViewModel;
+    variableValues: Record<string, TAllowedValues>;
```

Comment:
It will require a migration for Justin \ TNG's live data, but we really should improve the naming here.